ctapipe/containers.py

```diff
@@ -213,8 +276,8 @@ class ImageParametersContainer(Container):
     """ Collection of image parameters """
 
     container_prefix = "params"
-    hillas = Field(HillasParametersContainer(), "Hillas Parameters")
-    timing = Field(TimingParametersContainer(), "Timing Parameters")
+    hillas = Field(BaseHillasParametersContainer(), "Hillas Parameters")
```

Comment:
That does not make sense. This will be an incomplete container as default. Please use the containers with units degrees as default and add `type=BaseHillasParametersContainer`.

Reply:
Oh you just set a type to a field.
Yes, thats much better